packages/ui/src/views/chatflows/APICodeDialog.js

```diff
@@ -316,7 +318,8 @@ query(formData).then((response) => {
 `
         } else if (codeLang === 'cURL') {
             return `curl ${baseURL}/api/v1/prediction/${dialogProps.chatflowid} \\
-     -X POST \\${getConfigExamplesForCurl(configData, 'formData')}`
+     -X POST \\${getConfigExamplesForCurl(configData, 'formData')} \\
+     -H "Content-Type: application/json"`
```

Comment:
should be `application/x-www-form-urlencoded` when using `formData`

Reply:
thank you for pointing out this point!

I searched a bit and found we should use ` multipart/form-data` instead of `application/json` or `application/x-www-form-urlencoded`.
https://stackoverflow.com/questions/4007969/application-x-www-form-urlencoded-or-multipart-form-data

Reply:
have you tested it? is it not working if we dont specify anything there

Reply:
Yes I did!

what I did is 
- using `-H "Content-Type: application/json"` → failure
- using `-H "Content-Type: application/x-www-form-urlencoded"` → failure
- using `-H "Content-Type: multipart/form-data"` → succeed
- without Headers → succeed

current version of cURL use `multipart/form-data` as default when we use `-F` . But the reason I put `-H "Content-Type: multipart/form-data"` is defalut value can be changed by version update of cURL.

But if you think it too much, I will delete `-H "Content-Type: multipart/form-data"`

Reply:
ah okay thanks for testing different methods! its fine we can use `-H "Content-Type: multipart/form-data"` :)